### PR TITLE
Fixed links not working in next-steps.md

### PR DESCRIPTION
**Pull request by rcvalerio**
Fixed links not working in next-steps.md

### CODE REVIEW

**Comment by MaxHalford** [comment]
Weird, those links were working for me in development.

File: docs/introduction/next-steps.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Next steps
 
-The [Recipes 🍱](/recipes/reading-data) section is made up of small tutorials. Each one explains how to perform mundane tasks, such as measuring the performance of a model, selecting hyperparameters, etc.
+The [Recipes 🍱](latest/recipes/reading-data) section is made up of small tutorials. Each one explains how to perform mundane tasks, such as measuring the performance of a model, selecting hyperparameters, etc.
```
Comment (MaxHalford) [inline docs/introduction/next-steps.md:3]: Don't you have to preprend with a `/`?

**Review by MaxHalford — APPROVED** [review]

File: docs/introduction/next-steps.md (hunk 1 above)
Comment (rcvalerio) [reply docs/introduction/next-steps.md]: I did, thanks for pointing out.

**Review by MaxHalford — APPROVED** [review]

**Comment by MaxHalford** [comment]
Thanks a lot :)

**Pull request merged**